sandbox/optimal_args_hashbits.py

```diff
@@ -0,0 +1,122 @@
+#! /usr/bin/env python2
+#
+# This file is part of khmer, http://github.com/ged-lab/khmer/, and is
+# Copyright (C) Michigan State University, 2009-2015. It is licensed under
+# the three-clause BSD license; see doc/LICENSE.txt.
+# Contact: khmer-project@idyll.org
+#
+# pylint: disable=invalid-name,missing-docstring
+"""
+Estimate optimal arguments using hashbits counting.
+
+% python sandbox/optimal_args_hashbits.py  <data1> [ <data2> <...> ]
+
+Use '-h' for parameter help.
+"""
+from __future__ import print_function
+
+import sys
+import math
+import threading
+
+import khmer
+from khmer.khmer_args import build_hashbits_args
+from khmer.khmer_args import (report_on_config, info, add_threading_args)
+from khmer.kfile import check_input_files, check_space
+from khmer.kfile import check_space
+from oxli.functions import estimate_optimal_with_N_and_M
+from oxli.functions import estimate_optimal_with_N_and_f
+
+
+def get_parser():
+    parser = build_hashbits_args(descr="Load sequences into the compressible "
+                                 "graph format plus optional tagset.")
+    add_threading_args(parser)
+    parser.add_argument('input_filenames', metavar='input_sequence_filename',
+                        nargs='+', help='input FAST[AQ] sequence filename')
+    return parser
+
+def to_print_func(unique_kmers,fp_rate):
```

Comment:
Actually, rename and move to `oxli.functions`, and import on `sandbox/unique-kmers.py` and here.